### PR TITLE
feat(www): Add Community Plugin Badge

### LINKED ISSUE #21826: Add Community Plugin Badge
At the moment, Gatsby project maintained plugins are signified on the plugin listing page with an "official plugin" badge.
Official plugin badge

However, you need to notice this badge exists in order to recognize the absence of it on other pages.

It would be good to create a "Community Plugin" badge and associated text to use on all other plugin pages. This helps users understand the origin of the plugin and where it is maintained should they want to file an issue or PR.

### PR DESCRIPTION
**Pull request by akshitkrnagpal**
## Description

This PR adds Community Plugin Badge for all plugins that aren't official.

## Related Issues

Fixes #21826

### CODE REVIEW

File: www/src/components/package-readme.js [hunk 1]
```diff
@@ -45,35 +49,61 @@ const PackageReadMe = props => {
...
+                    display: `inline-block`,
+                    mr: 2,
+                  }}
+                  title={`Community Gatsby Plugin`}
+                >
+                  <CommunityIcon />
+                </span>
+                Community Plugin
+              </div>
+            )}
```
Comment (wardpeet) [inline www/src/components/package-readme.js]: Thank you for adding this icon 👍 

Can we abstract the component out of the page? Something along these lines? It makes it easier to read.

```js
const GatsbyPluginBadge = ({ isOfficial }) => {
  const Icon = isOfficial ? GatsbyIcon : CommunityIcon;
  const title = isOfficial
    ? "Official Gatsby Plugin"
    : "Community Gatsby Plugin";
  const text = isOfficial ? `Official Plugin` : `Community Plugin`;

  return (
    <div
      sx={{
        variant: `links.muted`,
        mr: 8,
        "&&": {
          border: 0,
          color: `textMuted`,
          display: `flex`,
          fontWeight: `body`
        },
        "&&:hover": {
          color: `textMuted`
        }
      }}
    >
      <span
        sx={{
          display: `inline-block`,
          mr: 2
        }}
        title={title}
      >
        <Icon />
      </span>
      {text}
    </div>
  );
};

```

```suggestion
              <GatsbyPluginBadge isOfficial={isOfficial} />
```

Comment (akshitkrnagpal) [reply www/src/components/package-readme.js]: > Thank you for adding this icon 👍

Looks like we might upgrade `react-icons` to 3.9. We will have this icon in that version. https://github.com/gatsbyjs/gatsby/pull/21820#issuecomment-592797020

> Can we abstract the component out of the page? Something along these lines? It makes it easier to read.

Sure. I'll do that.

Comment (wardpeet) [reply www/src/components/package-readme.js]: Just an inline component is fine :) Just put it at the top of this file.

> Looks like we might upgrade react-icons to 3.9. We will have this icon in that version. #21820 (comment)

Awesome!

**Comment by laurieontech** [comment]
Going to wait for #21859 and then we can update and get this merged!

**Comment by laurieontech** [comment]
@akshitkrnagpal #21860 was merged, so you should be good to update this!

**Comment by laurieontech** [comment]
This looks awesome! Thanks so much.

One of the things I noticed is that the "preview" of the plugin includes the Gatsby icon as well. Perhaps we should add the community icon?
Screen Shot 2020-03-25 at 12 51 21 PM

**Comment by akshitkrnagpal** [comment]
> One of the things I noticed is that the "preview" of the plugin includes the Gatsby icon as well. Perhaps we should add the community icon?

Nice catch. @laurieontech 😄 
I have updated the PR.

**Comment by laurieontech** [comment]
Wonderful, thank you!

**Pull request merged**